api/api.go

```diff
@@ -356,11 +374,27 @@ func (a *API) submitBlock(w http.ResponseWriter, r *http.Request) {
 	switch a.st.ForkVersion(a.st.HeadSlot()) {
 	case structs.ForkCapella:
 		var creq capella.SubmitBlockRequest
-		if err := json.NewDecoder(bytes.NewReader(b)).Decode(&creq); err != nil {
-			a.m.ApiReqCounter.WithLabelValues("submitBlock", "400", "payload decode").Inc()
-			writeError(w, http.StatusBadRequest, errors.New("invalid submitblock request capella decode"))
-			return
+		if r.Header.Get("Content-Type") == "application/octet-stream" {
```

Comment:
There is a chance we may like to do lowercase on that

Reply:
Look at the Go code for `http.Post`

<img width="890" alt="Screenshot 2023-05-18 at 11 33 40" src="https://github.com/blocknative/dreamboat/assets/28734408/15940d7e-3815-4fa1-90fd-12c3677fcb02">

Reply:
Yes, I see contentType is string ;> and ? ;)

Reply:
Capital too, not lower case